### PR TITLE
Use jQuery for all screen elements [WIP]

### PR DESCRIPTION
**Pull request by axic**
I had this lying around for a while.

### CODE REVIEW

**Comment by yann300** [comment]
tests are failing

**Comment by axic** [comment]
> tests are failing

It also depends on #452. We'll fix once that is merged.

File: src/app/compiler.js [hunk 1]
```diff
@@ -153,10 +155,11 @@ function Compiler (handleImportCall) {
       compilationFinished({error: 'Compiler not yet loaded.'})
     })
 
-    var newScript = document.createElement('script')
```
Comment (chriseth) [inline src/app/compiler.js]: I remember there might have been some magic involved and I specifically did not use jquery here, but I might also be mistaken.

Comment (axic) [inline src/app/compiler.js]: A very similar piece of code is used here too: https://github.com/ethereum/browser-solidity/blob/master/src/app.js#L212

It seems to work for me on Chrome, though that doesn't mean it works everywhere.

Comment (chriseth) [inline src/app/compiler.js]: It seems that jquery removes script tags from the dom after they were executed. So this is the difference between the two methods, but I don't know if that matters. Can you test the code with firefox?
The iframe example cannot be compared to this, I think.

**Comment by axic** [comment]
@yann300 @ninabreznik @serapath I'm not sure how relevant this is now,  but I think it would be nice to be consistent. Can you review, update or close?

**Comment by axic** [comment]
This is obsolete now.

**Pull request closed without merging**